src/main/component/Editor.svelte

```diff
@@ -82,24 +87,36 @@
   let editorRef: TinyMCEEditor | null;
   let lastVal = value;
   let disablindCache = disabled;
+  let readonlyCache = readonly;
   
+  const setReadonly = (editor: TinyMCEEditor, readonlyValue: boolean) => {
+    if (typeof editor.mode?.set === 'function' && isDisabledOptionSupported(editor)) {
+      editor.mode.set(readonlyValue ? 'readonly' : 'design');
+    }
+  }
+
+  const setDisabled = (editor: TinyMCEEditor, disabledValue: boolean, readonlyValue: boolean) => {
+    if (isDisabledOptionSupported(editor)) {
+        editor.options.set('disabled', disabledValue);
+      } else {
+        editor.mode.set(readonlyValue ? 'readonly' : 'design');
```

Comment:
I thought we decided that setting `disabled=true` will not trigger readonly mode on editor < 7.6

Reply:
I thought that we wanted to preserve the old behavior for editor < 7.6

Reply:
I might misread the comments above. Sorry about that. 

Yes we want to preserve the old behaviour of the editor < 7.6